UPTEthereumSigner/UPTEthSigner.h

```diff
@@ -30,23 +27,39 @@ typedef void (^UPTEthSignerDeleteKeyResult)(BOOL deleted, NSError *error);
 
 @class VALValet;
 
-@interface UPTEthereumSigner : NSObject
+@interface UPTEthSigner : NSObject
```

Comment:
The `UPTEthereumSigner` class is part of the main API of this library and renaming it seems like an unnecessary breaking change that will force downstream developers to refactor.
Is there a reason for renaming it that I'm not aware of?